.gitignore

```diff
@@ -10,3 +10,11 @@ Session.vim
 .netrwhist
 .idea
 *manifest-tool
+
+# Intellij generated files
+*.iml
+
+# emacs generated files
+.\#*
```

Comment:
Can you please not ignore emacs auto-saved files? It's useful to see these files to be aware that the added files do not contain the latest changes.